.devcontainer/setup_container.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Link root to new batman user
+rm /home/batman/.bashrc
+ln -s /root/.pyenv      /home/batman/.pyenv
+ln -s /root/.pyenvrc    /home/batman/.pyenvrc
+ln -s /root/.bashrc     /home/batman/.bashrc
+ln -s /root/.Xauthority /home/batman/.Xauthority
+
+# Setup permissions
+sudo chown batman: /root
+sudo chmod u+w /root
+sudo chown batman: /tmp/scons_cache
```

Comment:
add `-R`, otherwise scons won't be able to interact with cache, if it was previously used with root